docker-compose.yml

```diff
@@ -2,7 +2,8 @@ services:
   # Dependencies
   postgres:
     # TODO: Enable pgaudit on dev instance. See https://github.com/pgaudit/pgaudit/issues/44#issuecomment-455090262
-    image: postgres:15
+    # Includes wal2json
+    image: us-east1-docker.pkg.dev/firezone-staging/firezone/postgres:15
```

Comment:
Should we name it `postgres:15-waltojson`? Or `postgres-wal-to-json:15`?

Reply:
Hmm I'm not sure. We probably also need to build the `pgaudit` extension in here, and possibly more customizations in the future, which could make tagging ambiguous.